### DISCUSSION TITLE
Constrain values for path parameters

### DISCUSSION THREAD

**Discussion by abrevet-dev** [schemathesis/schemathesis#1642] [Q&A]
Hi,

I am looking for a generic way to constain the injected path parameters for all tests. Example: disable the character '%00'.

Thanks

**Comment by Stranger6667** [comment]
I know that it is super-outdated, but Schemathesis now ships a feature specifically to address this. Please, see [this](https://schemathesis.readthedocs.io/en/stable/data-generation.html#generating-strings) doc section for the details:

```
$ st run --generation-allow-x00=false ...
```
